models/maat/R/model2netcdf.MAAT.R

```diff
@@ -126,13 +131,20 @@ model2netcdf.MAAT <- function(rundir, outdir, sitelat = -999, sitelon = -999, st
                           seq(strptime(sub.maat.dates[1], format = "%Y-%m-%d", tz=timezone), by = "days", length = 1)))
       timestep.s <- day_secs / dims[1] # e.g. 1800 = 30 minute timesteps
       dayfrac <- 1 / dims[1]
-      day.steps <- seq(0, 0.99, 1 / dims[1])
+      day.steps <- head(seq(0, 1, by = dayfrac), -1)
       
       # setup netCDF time variable for year
       maat_run_start_by_year <- format(lubridate::as_datetime(sub.maat.dates, tz =timezone)[1], "%Y-%m-%d %H:%M:%S")
+      tvals <- (sub.maat.doy-1) + day.steps
+      bounds <- array(data=NA, dim=c(length(tvals),2))
```

Comment:
```suggestion
      bounds <- array(data = NA_real_, dim = c(length(tvals), 2))
```

Reply:
@ashiklom what does this do that the original didn't.  Just confirm the correct type?

Reply:
Yeah. In general, better to be explicit about types. `NA` is type logical, which can have some wonky behavior in edge cases. For instance, see [Exercise 2 in the Advanced R subsetting chapter](https://adv-r.hadley.nz/subsetting.html#exercises-9).